lib/adviser_list_csv.rb

```diff
@@ -5,11 +5,30 @@ def initialize(advisers)
 
   def to_csv(_ = {})
     CSV.generate do |csv|
-      csv << ['Ref. Number', 'Name', 'Firm']
-
+      csv << headings
       @advisers.each do |adviser|
-        csv << [adviser.reference_number, adviser.name, adviser.firm.registered_name]
+        csv << row(adviser)
       end
     end
   end
+
+  private
+
+  def headings
+    ['Ref. Number', 'Name', 'Firm', 'Accreditations', 'Qualifications']
```

Comment:
Could we use `%w` rather than a literal array syntax here please? 
https://github.com/moneyadviceservice/development-standards/tree/master/coding-styles/ruby#11.3

Should this array also be frozen?

Reply:
👍

Reply:
@willhall88 is `%w` still preferred when I need to escape the space in `Ref. Number`?

Reply:
ah, my bad. Missed that one. ok, happy as it currently is. But it still looks to me like it should be a frozen array?

Reply:
No worries, just thought I'd check what the preference is! Yep I'll change that 👍